packages/block-editor/src/components/editor-styles/index.js

```diff
@@ -80,6 +94,15 @@ export default function EditorStyles( { styles } ) {
 			{ transformedStyles.map( ( css, index ) => (
 				<style key={ index }>{ css }</style>
 			) ) }
+			<svg
+				xmlns="http://www.w3.org/2000/svg"
+				viewBox="0 0 0 0"
+				width="0"
+				height="0"
+				focusable="false"
+				role="none"
+				dangerouslySetInnerHTML={ { __html: transformedSvgs } }
+			/>
```

Comment:
Since we're adding all this front matter in this case and we're outputting them all at once in the frontend too, there's another performance improvement that we can make with duotone filters by wrapping all the filter definitions in a single SVG wrapper like this.

Not for this PR, but as a follow-up.

Reply:
Why would that be more performant?

Reply:
Re-reading what I wrote, I really didn't explain that well. What I'm saying is that we can have one wrapper for all the SVG `defs`, so `transformedSvgs` would just contain the definitions.

```xml
<filter id="wp-duotone-grayscale">
	<feColorMatrix ... />
	<feComponentTransfer ...>...</feComponentTransfer>
	<feComposite ... />
</filter>
<filter id="wp-duotone-midnight">
	...
</filter>
...
```

And the final output would look like this.

```xml
<svg xmlns="http://www.w3.org/2000/svg" ... />
	<defs>
		<filter id="wp-duotone-grayscale">
			<feColorMatrix ... />
			<feComponentTransfer ...>...</feComponentTransfer>
			<feComposite ... />
		</filter>
		<filter id="wp-duotone-midnight">
			...
		</filter>
		...
	</defs>
</svg>
```

Instead of what we currently have in this PR.

```xml
<svg xmlns="http://www.w3.org/2000/svg" ... />
	<svg xmlns="http://www.w3.org/2000/svg" ... />
		<defs>
			<filter id="wp-duotone-grayscale">
				<feColorMatrix ... />
				<feComponentTransfer ...>...</feComponentTransfer>
				<feComposite ... />
			</filter>
		</defs>
	</svg>
	<svg xmlns="http://www.w3.org/2000/svg" ... />
		<defs>
			<filter id="wp-duotone-midnight">
				...
			</filter>
		</defs>
	</svg>
	...
</svg>
```

In the frontend, doing something similar could maybe shave off another kilobyte if there are a lot of filters on the page.